test/run_tests

```diff
@@ -7,7 +7,7 @@ else
 fi
 if [ -z "$*" ]
 then
-    PYTHONPATH=$DIR exec $PYTHON ${DIR}/scapy/tools/UTscapy.py -t regression.uts -f html -l -o /tmp/scapy_regression_test_$(date +%Y%m%d-%H%M%S).html
+    PYTHONPATH=$DIR exec $PYTHON ${DIR}/scapy/tools/UTscapy.py -t regression.uts -f html -l -o /tmp/scapy_regression_test_$(date +%Y%m%d-%H%M%S).html -K mock_read_routes6_windows
```

Comment:
Yes: trying to load winpcapy under Linux will return an exception because it cannot load libpcap, and I didn't feel to add tons of crapy mocks to avoid loading it..